sdk/management/samples/src/main/java/com/azure/management/appservice/samples/ManageLinuxWebAppStorageAccountConnection.java

```diff
@@ -86,7 +86,7 @@ public static boolean runSample(Azure azure) {
 
             System.out.println("Creating web app " + app1Name + "...");
 
-            // FIXME the env variable will not work in linux since dot is not allowed in env variable name
+            // TODO: the env variable will not work in linux since dot is not allowed in env variable name
```

Comment:
I think you can change the TODO to just a note. I've updated the WAR to fallback to env var with "_" if "." not found.

Reply:
Done